sqrl-server/sqrl-server-vertx/src/test/java/com/datasqrl/graphql/WriteTest.java

```diff
@@ -142,7 +142,7 @@ private RootGraphqlModel getCustomerModel() {
                     .parentType("Query")
                     .fieldName("customer")
                     .match(ArgumentSet.builder()
-                            .query(new JdbcQuery("SELECT customerid FROM Customer", List.of()))
+                            .query(new JdbcQuery("postgres","SELECT customerid FROM Customer", List.of()))
```

Comment:
I'm wondering if we can use a static here like engine_name instead of typing postgres every time.